src/controllers/pages/student.js

```diff
@@ -100,7 +86,7 @@ export default class StudentController{
         }
         let today = moment();
         let dueDateMoment = moment.tz(dueDate, 'America/Chicago');
-        if(dueDateMoment.isBefore(today)){
+        if(!dueDateMoment.isAfter(today)){
```

Comment:
It was returning `false` for a `dueDateMoment` of today. We want it to return `true`, because "today" shouldn't be allowed as the due date of a longterm checkout.